front/poke/temporal/activities.ts

```diff
@@ -436,7 +436,7 @@ export async function deleteMembersActivity({
       transaction: t,
     });
 
-    const memberships = await Membership.findAll({
+    const memberships = await MembershipModel.findAll({
```

Comment:
this breaks the condition below (== 1). Let's use the activeMembership Resource function no?

Reply:
Went with `latestMembership` to keep behaviour ISO with now.